Source/Scene/Cesium3DTileset.js

```diff
@@ -0,0 +1,1821 @@
+         * Assign <code>undefined</code> to remove the style, which will restore the visual
+         * appearance of the tileset to its default when no style was applied.
+         * </p>
+         * <p>
+         * The style is applied to a tile before the {@link Cesium3DTileset#tileVisible}
+         * event is raised, so code in <code>tileVisible</code> can manually set a feature's
+         * properties (e.g. color and show) after the style is applied. When
+         * a new style is assigned any manually set properties are overwritten.
+         * </p>
+         *
+         * @memberof Cesium3DTileset.prototype
+         *
+         * @type {Cesium3DTileStyle}
+         *
+         * @default undefined
+         *
+         * @example
+         * tileset.style = new Cesium.Cesium3DTileStyle({
+         *    color : {
+         *        conditions : [
+         *            ['${Height} >= 100', 'color("purple", 0.5)'],
+         *            ['${Height} >= 50', 'color("red")'],
+         *            ['true', 'color("blue")']
+         *        ]
+         *    },
+         *    show : '${Height} > 0',
+         *    meta : {
+         *        description : '"Building id ${id} has height ${Height}."'
+         *    }
+         * });
+         *
+         * @see {@link https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/Styling|3D Tiles Styling language}
+         */
+        style : {
+            get : function() {
+                return this._styleEngine.style;
+            },
+            set : function(value) {
+                this._styleEngine.style = value;
+            }
+        },
+
+        /**
+         * The maximum screen space error used to drive level of detail refinement.  This value helps determine when a tile
+         * refines to its descendants, and therefore plays a major role in balancing performance with visual quality.
+         * <p>
+         * Depending on the tileset, <code>maximumScreenSpaceError</code> may need to be tweaked to achieve the right balance.
+         * Higher values provide better performance but lower visual quality.
+         * </p>
+         *
+         * @memberof Cesium3DTileset.prototype
+         *
+         * @type {Number}
+         * @default 16
+         *
+         * @exception {DeveloperError} <code>maximumScreenSpaceError</code> must be greater than or equal to zero.
+         */
+        maximumScreenSpaceError : {
+            get : function() {
+                return this._maximumScreenSpaceError;
+            },
+            set : function(value) {
+                //>>includeStart('debug', pragmas.debug);
+                Check.typeOf.number.greaterThanOrEquals('maximumScreenSpaceError', value, 0);
+                //>>includeEnd('debug');
+
+                this._maximumScreenSpaceError = value;
+            }
+        },
+
+        /**
+         * The maximum amount of GPU memory (in MB) that may be used by the tileset. This value is estimated from
```

Comment:
Instead of `may be user by the tileset`, which you contradict below with the SSE explanation, perhaps `may be used to cache tiles`?